db/migrate/20150911195321_add_budget_roles.rb

```diff
@@ -0,0 +1,24 @@
+class AddBudgetRoles < ActiveRecord::Migration
+  def change
+    User.with_email_role_slug!(
```

Comment:
I expect that a migration will only ever run once in any given environment, and that `db:seed` would never be run on an existing, populated database. If the garden is already growing, you don't seed it. It's for starting from scratch (as when running the rspec suite, which runs DatabaseCleaner and then loads the seeds to initialize a clean db instance on each run).

IME there are two kinds of migrations: (1) schema and (2) data. Most often it is (1) as we alter the db definitions, but occasionally it's (2) as we need to move existing data from one structure to another as the architecture changes.

I'd be happy to pull the relevant method out of `app/model` and put it in `lib/` if that's the objection here.